.github/workflows/tests.yml

```diff
@@ -104,6 +104,10 @@ jobs:
       - name: Run unit tests
         run: go test -v -race  ./...
 
+      - name: Run unit tests without CGO
+        # keep tags in sync with BUILDTAGS_CROSS in https://github.com/containers/podman/blob/2981262215f563461d449b9841741339f4d9a894/Makefile#L85
+        run: CGO_ENABLED=0 go test -tags "containers_image_openpgp exclude_graphdriver_btrfs exclude_graphdriver_devicemapper exclude_graphdriver_overlay" ./...
```

Comment:
Needs OpenSSL in the install step above.

Reply:
Nice catch, thanks! This should be fixed now.